CHANGELOG.md

```diff
@@ -82,6 +82,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Moved `compas.numerical.matrices` to `compas.topology.matrices`.
 * Moved `compas.numerical.linalg` to `compas.geometry.linalg`.
 * Changed `watchdog` dependency to be only required for platforms other than `emscripten`.
+* Changed `compas.geometry.earclip_polygon` algorithm because the current one does not work in several cases.
+* Changed the docstrings in `compas.geometry.earclip_polygon` are corrected. The orientation to the XY Frame has been removed since it is handled by the `polygon.to_vertices_and_faces` method. Additionally, the test_polygon.py file has been modified with a new test for the ear_clip called `test_polygon_to_vertices_and_faces`. The `object` has been added to all `Ear` classes.
+* Changed the docstrings in `compas.geometry.earclip_polygon` is merged back, meaning the polygon is transformed to XY frame.
 
```

Comment:
Don't need to add tests and docstring stuff to the changelog, the changelog is only for users of the library that need to understand what has changed